engine/class_modules/sc_death_knight.cpp

```diff
@@ -5532,6 +5545,9 @@ struct obliterate_strike_t : public death_knight_melee_attack_t
 
     trigger_icecap( execute_state );
 
+    if ( p() -> conduits.eradicating_blow->ok() )
+      p() -> buffs.eradicating_blow -> trigger();
```

Comment:
Even though this a single statement block please wrap it in braces to ensure consistency across the entire file.  Rule #1 of style guides is it doesn't matter what the style guide says, just that its consistent and followed.

Reply:
Updated as per suggestion

Reply:
I'm pretty sure the death knight file doesn't have consistent wrapping around single statement blocks, if it's still readable without wrapping, you don't need to force it 🤷

Reply:
I have no particular preference and had seen both used in the file, so I went with less lines of code originally.  If someone has a strong preference for wrapping everything though, I have no objections to doing so.